chart/infra-server/static/flavors.yaml

```
@@ -771,6 +771,28 @@
       value: "4"
       kind: optional
 
+    - name: aws-region
+      description: aws region for cluster
+      value: "us-east-2"
+      kind: optional
+
+    - name: managed-cp
+      description: Use managed control-plane?
+      value: "false"
+      kind: optional
+      help: |
+        Run as ROSA HCP?
+
+    - name: rosa-args
+      description: additional rosa cli args
+      value: ""
+      kind: optional
+      help: |
+        Arguments such as,
+        --sts
+        --private-link
+        See `rosa create cluster --help`
```

Help is now rendered as markdown, so this might be prettier
```suggestion
        Arguments such as,
        - `--sts`
        - `--private-link`
        See `rosa create cluster --help`
```